src/pages/Graph/GraphHelpFind.tsx

```diff
@@ -176,7 +176,8 @@ export default class GraphHelpFind extends React.Component<GraphHelpFindProps> {
       ['http > 0.5', `edges with http rate > 0.5 rps`],
       ['rt > 500', `edges with response time > 500ms. (requires response time edge labels)`],
       ['%httptraffic >= 50.0', `edges with >= 50% of the outbound http request traffic of the parent`],
-      ['node = svc and svc startswith det or !traffic', 'service node starting with "det" or edges with no traffic']
+      ['node = svc and svc startswith det or !traffic', 'service node starting with "det" or edges with no traffic'],
+      ['rank <= 2', 'top 2 highest ranking nodes']
```

Comment:
Not sure this is quite right.  It's not guaranteed to highlight 2 nodes.  Shouldn't it be something like, "nodes assigned a top 2 ranking"?

Reply:
Yes that's confusing. It's the "top 2" in terms of rank which could be more than 2 nodes and not the number 1 and number 2 nodes.